### ISSUE TITLE
Simple example fails.

### ISSUE THREAD

**Issue by jarrettj** [nozaq/terraform-aws-secure-baseline#70]
Hi,

Good day.

I get the following error for the simple example:
```
Error: Error describing organization: AWSOrganizationsNotInUseException: Your account is not a member of an organization.
	status code: 400, request id: 3e449e5a-082f-4c16-99a4-65ef6cdb07d8

  on .terraform/modules/secure-baseline_example_simple/nozaq-terraform-aws-secure-baseline-bb9cc79/bucket.tf line 40, in data "aws_organizations_organization" "org":
  40: data "aws_organizations_organization" "org" {}
```

In my main.tf:
```
module "secure-baseline_example_simple" {
  source  = "nozaq/secure-baseline/aws//examples/simple"
  version = "0.16.0"
  access_key = "qwer"
  audit_s3_bucket_name = "zxcv"
  secret_key = "asdf"
}
```

Regards.

**Comment by nozaq** [comment]
Hi @jarrettj , thanks for reporting it.
It seems like the module tried to read AWS Organization setting even when it is not explicitly directed (it should read it only when `account_type` argument is set to "master" or "member").

I've pushed a fix(#71) and released a new version(0.16.1). Could you try with it?

**Comment by jarrettj** [comment]
Thanks for the quick response! Did not expect that. It works :).
